scripts/sign.py

```diff
@@ -16,63 +16,156 @@ def int_parse(str):
 
 def get_args():
     from argparse import ArgumentParser
-
-    parser = ArgumentParser()
+    command_choices = ['sign', 'generate-digest', 'stitch-ta']
+
+    parser = ArgumentParser(
+        description='Sign a Tusted Application for OP-TEE.',
+        usage='\n   %(prog)s command [ arguments ]\n\n'
+
+        '   command:\n' +
+        '     sign             Generate signed loadable TA image file.\n' +
+        '                      Takes arguments --uuid, --in, --out' +
+        ' and --key.\n' +
+        '     generate-digest  Generate loadable TA binary image digest' +
+        ' for offline\n' +
+        '                      signing. Takes arguments  --uuid, --in and' +
+        ' --out.\n' +
+        '     stitch-ta        Generate loadable signed TA binary image' +
```

Comment:
Here also I prefer for `stitch` because (1) shorter and (2) why mention the TA and not the signature? It's two things that you are stitching together...